hnn_core/parallel_backends.py

```diff
@@ -65,6 +65,8 @@ def _gather_trial_data(sim_data, net, n_trials, postproc):
         net.cell_response.update_types(net.gid_ranges)
         net.cell_response._vsoma.append(spikedata[3])
         net.cell_response._isoma.append(spikedata[4])
+        for e_idx in range(len(net.lfp)):
+            net.lfp[e_idx]['data'].append(spikedata[5][e_idx])
```

Comment:
Maybe change `spike_data` to `sim_data_trial` since that variable name doesn't really make sense any more.